app/apps/workflow/views.py

```diff
@@ -172,6 +176,11 @@ def get_district(self, queryset, name, value):
             )
         return queryset
 
+    def get_housing_corporation(self, queryset, name, value):
+        if value:
+            return queryset.filter(case__address__housing_corporation__in=value)
```

Comment:
Hier zou je ergens moeten afdwingen dat value een type list is.
Als het bijvoorbeeld een string wordt krijg je gekke dingen.
verder is value beetje nietszeggend, wat is het, een adres of zo ?